doc/manual/expressions/builtins.xml

```diff
@@ -280,8 +280,15 @@ with import (fetchTarball https://github.com/NixOS/nixpkgs-channels/archive/nixo
 
 stdenv.mkDerivation { … }
 </programlisting>
-
-    Note that when obtaining the hash with <varname>nix-prefetch-url
+    <para>The fetched tarball is chached for a certain amount of time 
+    (1 hour by default) in <literal>~/.cache/nix/tarballs/</literal>.
+    You can change the cache timeout either in command line with 
+    <literal>--option tarball-ttl <number of seconds></literal> or
```

Comment:
`<number of seconds>` is not valid XML. You should do something like `<replaceable>ttl</replaceable>`.